advanced_source/static_quantization_tutorial.py

```diff
@@ -306,7 +306,33 @@ def print_size_of_model(model):
 # As our last major setup step, we define our dataloaders for our training and testing set.
 # The specific dataset we've created for this tutorial contains just 1000 images, one from
 # each class (this dataset, at just over 250 MB, is small enough that it can be downloaded
-# relatively easily). These functions mostly come from
+# relatively easily). The URL for this custom dataset is:
```

Comment:
We should also add information on how one can download imagenet for pytorch here. We could say that:
If you want to download the entire imagenet dataset, please follow instructions at:

https://pytorch.org/docs/stable/torchvision/datasets.html#imagenet